build.sbt

```diff
@@ -39,6 +39,8 @@ lazy val lambda = (project in file("lambda"))
       awsSQS,
       awsStateMachine,
       http,
+      circeGeneric,
+      circeParser,
```

Comment:
We're already using [upickle](https://com-lihaoyi.github.io/upickle/) as a json-handling library.